### PR TITLE
Feature/more archiver

### PR DESCRIPTION
**Pull request by samchrisinger**

### CODE REVIEW

File: website/project/utils.py [hunk 1]
```diff
@@ -1,16 +1,11 @@
...
 """Various node-related utilities."""
-from website.project.views import node as node_views
-
 from website import mails
 from website import settings
 
 from website.util.permissions import ADMIN
 
-# Alias the project serializer
-serialize_node = node_views._view_project
```
Comment (sloria) [inline website/project/utils.py]: Why was this removed? 

This was added because the name and location (utils.py) make more sense than `_view_project`. `_view_project` still exists just for backwards compat.

Comment (samchrisinger) [inline website/project/utils.py]: I had a circular import in test_addons.py. @sloria does this look better now?
